includes/functions_install.php

```diff
@@ -136,10 +136,20 @@ function load_schema_31($install_path = '', $install_dbms = false)
 			include($phpbb_root_path . 'includes/constants.' . $phpEx);
 		}
 
-		$finder = new \phpbb\finder(new \phpbb\filesystem(), $phpbb_root_path, null, $phpEx);
+		// Check if equal or greater than 4.0@dev
+		if (defined('PHPBB_40'))
+		{
+			$finder = new \phpbb\finder($phpbb_root_path, null, $phpEx);
+		}
+		else
+		{
+			$finder = new \phpbb\finder(new \phpbb\filesystem(), $phpbb_root_path, null, $phpEx);
+		}
+
 		$classes = $finder->core_path('phpbb/db/migration/data/')
 			->get_classes();
 
+		// Check if greater than 3.2
```

Comment:
Please remove this comment (and you can also remove the one above, it's not necessary as the test for PHPBB_40 is self-explanatory)